bin/export.py

```diff
@@ -321,6 +321,13 @@ def update_problems_yaml(problems):
                 }
             )
 
+    if colors:
+        for d, c in zip(data, colors):
```

Comment:
Add comment/doc that order is as in problems.yaml, and check the lists have the same length to be sure.